From: Philip Withnall <pwithnall@endlessos.org>
Date: Mon, 6 Mar 2023 14:38:58 +0000
Subject: gresourcefile: Fix a missing newline in GResourceFile

This introduces no functional changes.

Signed-off-by: Philip Withnall <pwithnall@endlessos.org>
(cherry picked from commit 303d79989cfd2c4ee265380d3f0ec3731f6a63e2)
---
 gio/gresourcefile.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/gio/gresourcefile.c b/gio/gresourcefile.c
index 24f20f2..488fa34 100644
--- a/gio/gresourcefile.c
+++ b/gio/gresourcefile.c
@@ -558,7 +558,8 @@ g_resource_file_query_filesystem_info (GFile         *file,
   if (g_file_attribute_matcher_matches (matcher, G_FILE_ATTRIBUTE_FILESYSTEM_TYPE))
     g_file_info_set_attribute_string (info, G_FILE_ATTRIBUTE_FILESYSTEM_TYPE, "resource");
 
-  if (g_file_attribute_matcher_matches (matcher, G_FILE_ATTRIBUTE_FILESYSTEM_READONLY))    g_file_info_set_attribute_boolean (info, G_FILE_ATTRIBUTE_FILESYSTEM_READONLY, TRUE);
+  if (g_file_attribute_matcher_matches (matcher, G_FILE_ATTRIBUTE_FILESYSTEM_READONLY))
+    g_file_info_set_attribute_boolean (info, G_FILE_ATTRIBUTE_FILESYSTEM_READONLY, TRUE);
 
   g_file_attribute_matcher_unref (matcher);
 
